css/excercise_one/excercise_one.html

```diff
@@ -0,0 +1,111 @@
+      </a>
+      <div class="nav-link">
+        <a href="#" class="small-light-text">Características</a>
+        <a href="#" class="small-light-text">Precio</a>
+        <a href="#" class="small-light-text">Contacto</a>
+      </div>
+      <div class="nav-register">
+        <a href="#" class="small-light-text">Ingresar</a>
+        <a href="#" class="sign-up link-border">Registrarse gratis</a>
+      </div>
+    </nav>
+    <div class="home">
+      <div class="main-page">
+        <div class = "main-page-corner">
+          <img src="../../img/group 2384.png" alt="Logo" class="corner-logo">
+          <p class="corner-title">Empresas</p>
+        </div>
+        <div>
+          <h2 class="main-phrase">Generá acciones<br> que <span class="main-phrase-second-part">sorprendan a<br> tu equipo y clientes</span></h2>
+        </div>
+        <div class="home-buttons">
+          <button class="register-button register-text-button">
+            REGISTRATE ACÁ
+          </button>
+          <button class="video-button video-text-button">
+            VER VIDEO TUTORIA
+          </button>
+        </div>
+        <div class="users-corner">
+          <div class="user-photo-background">
+            <img src="../../img/Union.png" alt="User" class="user-photo">
+          </div>
+          <p class="desired-users">Para personas &gt;</p>
+        </div>
+        <div class="planIT">
+          <p>En PlanIT también podés encontrar regalos para<br> tu familia y amigos.</p>
+        </div>
+      </div>
+      <div class="gallery">
+        <div class="gallery-col">
+          <div class="card">
+            <img src="../../img/eventos.png" alt="Eventos" class="events-photo">
+            <div class="text-boxes">
+              <p class="card-text">Eventos</p>
+            </div>
+          </div>
+          <div class="card">
+            <img src="../../img/regalos.png" alt="Regalos" class="gifts-photo">
+            <div class="text-boxes">
+              <p class="card-text">Regalos<br> personalizados</p>
+            </div>
+          </div>
+        </div>
+        <div class="gallery-col">
+          <div class="card">
+            <img src="../../img/catering.png" alt="Catering" class="left-photos">
+            <div class="text-boxes">
+              <p class="card-text">Catering</p>
+            </div>
+          </div>
+          <div class="card">
+            <img src="../../img/merchandising.png" alt="Merchandising" class="left-photos">
+            <div class="text-boxes">
+              <p class="card-text">Merchandising</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="clients-logos">
+      <div class="row-companies">
+        <p class="clients-phrase">Trabajamos día a día junto a empresas que<br> confían en nosotros</p>
+      </div>
+      <div class="row-companies">
+        <img class="img-company-row" src="../../img/Basf.png" alt="BASF">
```

Comment:
in cases like this when you have a div with many images inside (that all serve the same purpose) you could have a class on the div instead of repeating it on every image (just to have a cleanear code)
```
.image-div (just an example name) {
   img {
      styles for image here
   }
}
```